hazelcast/src/test/java/com/hazelcast/map/InterceptorTest.java

```diff
@@ -98,28 +99,33 @@ public void testMapInterceptor() throws InterruptedException {
     }
 
     @Test
-    public void testMapInterceptorOnNewMember() throws InterruptedException {
-        TestHazelcastInstanceFactory nodeFactory = createHazelcastInstanceFactory(2);
+    public void testMapInterceptorOnNewMember() {
         Config config = getConfig();
-        HazelcastInstance instance1 = nodeFactory.newHazelcastInstance(config);
-        IMap<Integer, Object> map = instance1.getMap("map");
+        TestHazelcastInstanceFactory nodeFactory = createHazelcastInstanceFactory(2);
+        HazelcastInstance hz1 = nodeFactory.newHazelcastInstance(config);
+        IMap<Integer, Object> map1 = hz1.getMap("map");
+
         for (int i = 0; i < 100; i++) {
-            map.put(i, i);
+            map1.put(i, i);
         }
-        map.addInterceptor(new NegativeInterceptor());
+
+        map1.addInterceptor(new NegativeInterceptor());
         for (int i = 0; i < 100; i++) {
-            assertEquals(i * -1, map.get(i));
+            assertEquals("Expected negative value on map1.get(" + i + ")", i * -1, map1.get(i));
         }
-        HazelcastInstance instance2 = nodeFactory.newHazelcastInstance(config);
+
+        HazelcastInstance hz2 = nodeFactory.newHazelcastInstance(config);
+        IMap<Integer, Object> map2 = hz2.getMap("map");
         for (int i = 0; i < 100; i++) {
-            assertEquals(i * -1, map.get(i));
+            assertEquals("Expected negative value on map1.get(" + i + ")", i * -1, map1.get(i));
+            assertEquals("Expected negative value on map1.get(" + i + ")", i * -1, map2.get(i));
```

Comment:
Added this to test, that the interceptor is also available on the new node (although it has been added programmatically at runtime on the old node).